lib/tasks/mqtt.rake

```diff
@@ -0,0 +1,10 @@
+task mqtt: :environment do
+  MQTT::Client.connect('test.mosquitto.org') do |client|
+    client.get('rs/covid-detector/sensor/+/measure') do |topic, message|
```

Comment:
and maybe the same here. I also suggest to remove the sensor_id from the topic, and just publish on `rs/covid-detector/measure`, wdyt?